modules/prebidServerBidAdapter.js

```diff
@@ -196,10 +198,15 @@ function PrebidServer() {
             bidObject.creative_id = bidObj.creative_id;
             bidObject.bidderCode = bidObj.bidder;
             bidObject.cpm = cpm;
-            bidObject.ad = bidObj.adm;
-            if (bidObj.nurl) {
-              bidObject.ad += utils.createTrackPixelHtml(decodeURIComponent(bidObj.nurl));
+            if (bidObj.media_type === 'video') {
+              bidObject.vastUrl = bidObj.nurl;
```

Comment:
Is a `nurl` guaranteed on video bids?

In section `6.4.2` of the OpenRTB 2.5 spec, it looks like video bids could either serve `vastUrl` through the `nurl`, _or_ `vastXml` through `adm`.